package com.redhat.ceylon.compiler.java.test.statement.trycatch;

class E extends .ceylon.language.Exception {
    
    E(final .ceylon.language.String description, final .java.lang.Throwable cause) {
        super(description, cause);
    }
}
class TryCatchSubclass {
    
    TryCatchSubclass() {
    }
    
    private final void t$priv() {
    }
    
    private final void ce$priv(final .com.redhat.ceylon.compiler.java.test.statement.trycatch.E e) {
    }
    
    private final void cexception$priv(final .java.lang.Throwable e) {
    }
    
    private final void m$priv() {
        try {
            t$priv();
        } catch (final .com.redhat.ceylon.compiler.java.test.statement.trycatch.E e) {
            ce$priv(e);
        } catch (final .java.lang.Exception e) {
            cexception$priv(e);
        }
    }
    
    public static void main(.java.lang.String[] args) {
        .ceylon.language.process_.getProcess$().setupArguments(args);
        new .com.redhat.ceylon.compiler.java.test.statement.trycatch.TryCatchSubclass();
    }
}